requirements.txt

```diff
@@ -31,7 +31,7 @@ pytest-timeout>=1.3.0,<2.0.0
 mock>=2.0.0,<3.0.0
 flake8>=3.8.0,<3.10.0
 hypothesis>=3.27.0,<7.0.0
-mypy>=0.910
+mypy==0.910
```

Comment:
I'd prefer if this change isn't part of this PR. We can cherry-pick the commit from `master` into `develop` (or otherwise update `develop` from `master`) and then merge `upstream/develop` into this branch.

Reply:
Agreed! Let's update `develop` with `master` first, we'll need that fix for the CI either way.

Reply:
I can rebase it once it's in `develop`. Then it's completely vanished from the PR's history.

Reply:
`develop` has been updated, so if you pul/merge this should hopefully be resolved